packages/components/view-switcher/src/view-switcher.stories.tsx

```diff
@@ -0,0 +1,144 @@
+import type { ComponentProps } from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+import ViewSwitcher from '.';
+import type { TViewSwitcherButtonProps } from './view-switcher-button';
+import { iconArgType } from '@/storybook-helpers';
+
+const BUTTONS_COUNT = 4;
+const DEFAULT_BUTTON_ICON = 'WorldIcon';
+
+type ViewSwitcherPropsAndCustomArgs = ComponentProps<
+  typeof ViewSwitcher.Group
+> & {
+  iconViewSwitcherButton1: string;
+  iconViewSwitcherButton2: string;
+  iconViewSwitcherButton3: string;
+  iconViewSwitcherButton4: string;
+  iconOnlyButton1: boolean;
+  iconOnlyButton2: boolean;
+  iconOnlyButton3: boolean;
+  iconOnlyButton4: boolean;
+  isDisabledButton1: boolean;
+  isDisabledButton2: boolean;
+  isDisabledButton3: boolean;
+  isDisabledButton4: boolean;
+};
```

Comment:
This adheres to the pattern outlined [here](https://storybook.js.org/docs/writing-stories/typescript#typing-custom-args)
To the best of my knowledge, there is no method for grouping storybook v7 controls in a manner similar to what we are achieving with storybook v5 knobs e.g. [here](https://github.com/commercetools/ui-kit/blob/699175cc0e8d5b1b11419bde7e8c8fb48c753d47/packages/components/view-switcher/src/view-switcher-with-icons.story.js#L45)

Reply:
I think this is something we might try to improve after the migration.

Reply:
Something that might help here is to visually group the controls related to the items in the props table using this config:
```js
argTypes: {
  iconViewSwitcherButton1: {
    table: {
      subcategory: 'Items controls',
    }
  }
}
```


![image](https://github.com/commercetools/ui-kit/assets/97907068/1520ac8f-8cda-4cbd-a62c-3538b2cba106)

Reply:
Thank you for researching it 🙏 I pushed an update 8f2315e1d21d7a91bf31ed46a7150477bbe97b4c